capstone/src/main/java/com/google/sps/data/Form.java

```diff
@@ -15,7 +15,7 @@
     public Form(Entity entity) {
         this.entity = entity;
     }
-
+     
```

Comment:
Unnecessary whitespace. Also if you do want to make little formatting changes like this, its easier to lump them all up into one big formmatting CL full of meaningless changes so its easier to review. (because sadly github cant tell this is an irrelevant change). That helps it so that I can just whiz through a formatting CL in 2 seconds and it doesnt clutter CLs with actual content

Reply:
Yea this is my bad. I'll keep the meaningless changes CL thing in mind for future CLs.